charts/t8s-cluster/templates/management-cluster/clusterClass/kubeadmnControlPlaneTemplate/_kubeadmControlPlaneTemplateSpec.yaml

```diff
@@ -4,71 +4,74 @@ Here we are generating a hash suffix.
 This function needs the whole `$` context to be able to use `.Files.Get`
 */}}
 {{- define "t8s-cluster.clusterClass.kubeadmControlPlaneTemplate.spec" -}}
-clusterConfiguration:
```

Comment:
Not on the content, but the folder has a typo. 

`kubeadmnControlPlaneTemplate` instead of `kubeadmControlPlaneTemplate`.